src/components/Books.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Books = () => (
+  <main>
+    <div />
+    <h2 className="add-book-h2">ADD NEW BOOK</h2>
+    <form className="form">
+      <input className="input-title" type="text" placeholder="Book title" />
+      <input className="input-category" type="text" placeholder="Category" />
+      <button className="add-book-btn" type="submit">Add Book</button>
+    </form>
```

Comment:
- [ ] In keeping with the SOLID principle kindly extract this form into a new separate component and render that component in this **`Books`** component instead.

Reply:
This change has been done succesfully.